### PR TITLE
lots o changes

### PR DESCRIPTION
**Pull request by allie-rae**
## Updates:
- login links and logout links differentiated
- user options page
- new routes
- mostly empty swipe page (just a header)
- matches page w/ dummy matches
- login now has a loading button
- login now has an error message if the login fails
- login now navigates to user options on successful login
- logout navigates to home 
- new eslint rule
- styled links on login and register pages
- new import (don't forget to delete your node modules & reinstall!)

## User options page:
Screen Shot 2022-09-11 at 2 58 06 AM

## Dummy matches list page:
Screen Shot 2022-09-11 at 2 57 16 AM

## Find a match (just a header for now):
Screen Shot 2022-09-11 at 2 58 44 AM

## Login error:
Screen Shot 2022-09-11 at 2 59 55 AM

## Login loading button:
Screen Shot 2022-09-11 at 3 01 47 AM

### CODE REVIEW

**Review by jae-finger — APPROVED** [review]
Thanks for cleaning up the front end base code !

**Review by DeusExMashima — CHANGES_REQUESTED** [review]
I was wondering should the major changes like es-lint changes, that are involved in a lot of places in the code base such as typescript config should be discussed in a group before these changes are implemented. It causes confusion and I just want to make sure we consider the pros and cons as a group before major changes are made. 

Also, I'd like to discuss code standards if possible. I know this project is not things that will be used widely, but keeping things uniform and in an agreeable pattern will help with readability for the team, and make the code base as clean and as maintainable as possible

File: src/pages/Layout/Layout.tsx [hunk 1]
```diff
@@ -16,27 +16,35 @@ import { useNavigate } from 'react-router-dom';
...
   const { window, children } = props;
   const [mobileOpen, setMobileOpen] = React.useState(false);
   const navigate = useNavigate();
 
+  const logout = () => {
+    localStorage.removeItem('token');
+    navigate('/');
+  };
+
+  const loggedInNavItems = [
```
[the remark this answers is not in the document]
Comment (allie-rae) [reply src/pages/Layout/Layout.tsx:29]: Feel free to do that if you want to improve the code later :)

File: src/pages/Layout/Layout.tsx [hunk 2]
```diff
@@ -107,18 +106,15 @@ const Layout = (props: any) => {
               sniffr
             </Typography>
             <Box sx={{ display: { xs: 'none', sm: 'block' } }}>
-              {navItems.map((item: NavItem) => (
```
[the remark this answers is not in the document]
Comment (allie-rae) [reply src/pages/Layout/Layout.tsx:110]: Feel free to add that back in :)

File: src/pages/Layout/Layout.tsx [hunk 3]
```diff
@@ -16,27 +16,35 @@ import { useNavigate } from 'react-router-dom';
...
   const [mobileOpen, setMobileOpen] = React.useState(false);
   const navigate = useNavigate();
 
+  const logout = () => {
+    localStorage.removeItem('token');
+    navigate('/');
+  };
+
+  const loggedInNavItems = [
+    { label: 'Home', func: () => navigate('/user-options') },
```
[the remark this answers is not in the document]
Comment (allie-rae) [reply src/pages/Layout/Layout.tsx:30]: Function was necessary to include logout

File: src/pages/Matches/Matches.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+        <Table sx={{ width: '100%' }} aria-label="simple table">
+          <TableHead>
+            <TableRow>
+              <TableCell>Name</TableCell>
+              <TableCell>Owner</TableCell>
+              <TableCell>Email</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {dummyMatchRows.map(row => (
```
[the remark this answers is not in the document]
Comment (allie-rae) [reply src/pages/Matches/Matches.tsx:39]: Feel free to add in any interfaces you'd like

**Pull request merged**